lib/installed_clients/AssemblyUtilClient.py

```diff
@@ -7,6 +7,9 @@
 ############################################################
```

Comment:
This is a code generated file and should not be edited

Reply:
Got it. I'm on a call, but want to ask you about this as soon as I'm free.